docs/usage/configuration-options.md

```diff
@@ -448,6 +448,17 @@ Renovate's default behavior is to reuse/reopen a single Config Warning issue in
 However for some people this has the downside that the config warning won't be sorted near the top if you view issues by creation date.
 Configure this option to `false` if you prefer Renovate to open a new issue whenever there is a config warning.
 
+## conservative
```

Comment:
```suggestion
## bundlerConservative
```

I think better to name this specific to Bundler and not confuse people. Will also require this section to be moved to remain alphabetically ordered

Reply:
didn't we agreed to move this to `postUpdateOptions`. instead? 🤔

Reply:
Yeah, I forgot

Reply:
https://github.com/renovatebot/renovate/discussions/16128#discussioncomment-2973178
☝️

Reply:
Sorry; I just haven't had a chance to get back to this since that comment. I was out of town last week. Should be able to update today.